Project.toml

```
@@ -1,28 +1,30 @@
 name = "Elemental"
 uuid = "902c3f28-d1ec-5e7e-8399-a24c3845ee38"
-version = "0.5.1"
+version = "0.6.0"
 
 [deps]
 Distributed = "8ba89e20-285c-5b6f-9357-94700520ee1b"
 DistributedArrays = "aaf54ef3-cdf8-58ed-94cc-d582ad619b94"
-LibGit2 = "76f85450-5226-5b5a-8eaa-529ad045b433"
+Elemental_jll = "c2e960f2-a21d-557e-aa36-859d46eed7e8"
 Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 
 [compat]
 DistributedArrays = "0.5, 0.6"
+Elemental_jll = "< 1"
```

```suggestion
Elemental_jll = "0.87"
```
To avoid upstream breakage